docs/02.API-REFERENCE.md

```diff
@@ -4084,8 +4085,10 @@ jerry_to_property_descriptor (jerry_value_t obj_value, jerry_property_descriptor
 - `src_prop_desc_p` - the output property descriptor.
 - return
   - [jerry_value_t](#jerry_value_t)
-  - true - if success
-  - false - if error occurs
+  - true - if successful
+  - error - otherwise
+
```

Comment:
at other places we use the:
```
 - true, if success
 - thrown error, otherwise
```
syntax. In the future we should reword all places to a common format. This is ok for now.

Reply:
The function prototype above should also be changed.